packages/media/src/types.js

```diff
@@ -68,7 +68,7 @@ ResourcePropTypes.videoResource = PropTypes.shape({
   width: PropTypes.number.isRequired,
   height: PropTypes.number.isRequired,
   poster: PropTypes.string,
-  posterId: PropTypes.number,
+  posterId: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Where does it happen that the `posterId` is a string?

Reply:
I think it’s for 3rd party media since they are provided by coverr. Will update this comment with a screenshot.